mods/default/functions.lua

```diff
@@ -560,8 +560,13 @@ minetest.register_abm({
 	catch_up = false,
 	action = function(pos, node)
 		node.name = moss_correspondences[node.name]
-		minetest.set_node(pos, node)
-	end
+        if(node.name ~= nil) then
+            minetest.set_node(pos, node)
+        else
+            minetest.log("action", "[MOD]default: Wrong Node @Pos: " ..
+                         minetest.pos_to_string(pos) .. " tried to turn in mossy.")
```

Comment:
Does it really need to be logged? I'm not sure if mods can do much about it.

Reply:
Please don't add that to MTG. It will only spam the logs until the bug is fixed in the engine.

Reply:
I agree with SmallJoker.

Reply:
You're right, it isn't not so important, that it have to be logged. I thought there more on an developer for mods.